CHANGELOG_UNRELEASED.md

```diff
@@ -26,6 +26,52 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - in `interval.v`, new lemmas: `ge_pinfty`, `le_ninfty`, `gt_pinfty`, `lt_ninfty`.
 
+- in `order.v`
```

Comment:
Could this be a separate commit?

Reply:
How do you suggest to split? (They are related in cascade, each idem depends on the previous one)

Reply:
I was just thinking that the first 3 items were mostly independent of block matrices. 
Tagnat instead seems more closely related to matrices.